connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/RemoteSparkSession.scala

```diff
@@ -49,7 +49,7 @@ import org.apache.spark.util.Utils
 object SparkConnectServerUtils {
 
   // Server port
-  private[connect] val port = ConnectCommon.CONNECT_GRPC_BINDING_PORT + util.Random.nextInt(1000)
+  val port: Int = ConnectCommon.CONNECT_GRPC_BINDING_PORT + util.Random.nextInt(1000)
```

Comment:
Can we use a ephemeral port to make sure on the test flakiness? See also `SparkConnectService.localPort`, and https://github.com/apache/spark/blob/c7007b37e15ed1dc7c27e44576aa21f2e9e10e3f/python/pyspark/sql/connect/session.py#L503-L506

Reply:
Can be done in a separate PR.

BTW, you could use `private[spark] val port: Int` accessor